src/training/day4/SportCar.java

```diff
@@ -0,0 +1,8 @@
+package training.day4;
+
+public class SportCar extends Vehicle{
+     private final double DEFAULT_FUEL_CONSUMPTION = 10;
+    public SportCar(double fuel, int hp) {
```

Comment:
Not on the same indentation as the constant. Keep then on equal column. Consistency....